procedures/auxtel/observation_procedures/Parabolic_Focus_Determination_Data_Acq.ipynb

```diff
@@ -9,23 +9,42 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    from lsst.ts.externalscripts.auxtel.latiss_cwfs_align import LatissCWFSAlign
</pre><p>should use WEP align</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Already fixed.</p>